images/jupyter-user/bootstrap.sh

```diff
@@ -47,3 +47,11 @@ do
 done
 
 rm -fR /home/jovyan/.aws
+
```

Comment:
I don't think  "/home/jovyan/.local/share/code-server/CachedExtensionVSIXs" should ever exists on startup as this is a clean install.  Anyway , this check does not hurt I guess

Reply:
Other modules use this bootstrap so the check is necessary

Reply:
Looks like other people have the same issue:  https://github.com/cdr/code-server/issues/1140 (last comment)

Reply:
Modified to look for both the existence of the source dir and the symlinks

Reply:
Amir- you re correct.  I am moving this to the start script for codeserver...and reverting on next commit